src/baldr/compression_utils.cc

```diff
@@ -83,7 +83,7 @@ bool inflate(const std::function<void(z_stream&)>& src_func,
       if (stream.avail_in == 0)
         src_func(stream);
       if (stream.avail_in == 0)
-        throw;
+        throw std::exception();
```

Comment:
From the [docs](https://en.cppreference.com/w/cpp/language/throw), `This form is only allowed when an exception is presently being handled (it calls std::terminate if used otherwise). `

I think this fix is good :+1: